docs/en/latest/how-to-build.md

```diff
@@ -119,7 +119,18 @@ Follow the steps below to install Apache APISIX via the source release package.
 
   **Note**: If `make deps` fails with "install `lualdap` failed" with an error like `Could not find header file for LDAP` try the solution below.
 
-  Solution: Set `LDAP_DIR` with `luarocks config` manually. For example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`.
+      Solution: luarocks supports adding custom package filepaths to the project. You can set `LDAP_DIR` with `luarocks config` manually, for example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`. You should config as follows:
+      1. Install `openldap` locally using tools including but not limited to 'brew';
```

Comment:
Why do other Lua dependencies not have this issue?

Reply:
This is not just a dependency issue.
I have the same problem with PCRE & openssl.
This is a general solution.

Reply:
So I think you should write a generic method